host/src/components/QuestionSettingsCard.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
```

Comment:
Container component for settings on a per-question basis. New features will be added to this as they are availabile (like confidence and surfacing student thinking). Structuring this as a card, as that will be the format when integrated into the updated `host` app per the latest design